design/common/separate-git-repository-for-helm-charts.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Separate git Repository for Helm Charts
+target-version: release-1.6
+---
+
+# Separate git Repository for Helm Charts
+
+## Summary
+
+### Goals
+
+The goal of this design document / change is to use a separate git repository for Helm charts for the Rook ecosystem.
+The Rook ecosystem Helm charts include but not limited to the Rook operators and Rook custom resources.
+
+To know this goal has been achieved, a separate repository with the current Rook operator Helm charts and upcoming Helm charts should have been created.
+In addition to the repository, CI workflows should be established to be able to release the Helm charts when needed.
+
+### Non-Goals
+
+The design document is not about introducing new Helm Charts, but to move the Helm charts development to a separate repository.
+
+## Proposal details
+
+The proposal is to create a separate repository for the Helm charts.
+In the new repository for the Helm charts, a CI/CD workflow using GitHub Actions,
+utilizing the [GitHub chart-releaser Action](https://github.com/helm/chart-releaser-action) should be created to easily
```

Comment:
How would we track different releases of rook with the helm chart? We would have to branch the helm repo whenever we branch the rook repo, right?

Reply:
As of right now, my idea would be to only track the latest Rook release, allowing people to be able to automatically move forward with the Rook version.